### ISSUE TITLE
combat module crashed

### ISSUE THREAD

**Issue by kevin01523** [mrmin123/kancolle-auto#23]
```

[2015-10-01 16:32:11] At Home!
[2015-10-01 16:32:11] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 16:32:15] No, no fleets to receive!
[2015-10-01 16:32:15] Navigating to Sortie menu!
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(505,345)@S(0)[0,0 1360x768]
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[log] CLICK on L(539,315)@S(0)[0,0 1360x768]
[debug] waiting for combat_area_2.png to appear
[debug] combat_area_2.png has appeared.
[debug] waiting for combat_area_2.png to appear
[debug] combat_area_2.png has appeared.
[log] CLICK on L(539,517)@S(0)[0,0 1360x768]
[debug] waiting for combat_panel_2-3.png to appear
[debug] combat_panel_2-3.png has appeared.
[debug] waiting for combat_panel_2-3.png to appear
[debug] combat_panel_2-3.png has appeared.
[log] CLICK on L(566,426)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(991,520)@S(0)[0,0 1360x768]
[2015-10-01 16:32:31] Checking fleet condition...
[2015-10-01 16:32:40] Light damage: 0; moderate damage: 0; critical damage: 0
[2015-10-01 16:32:50] Commencing sortie!
[debug] waiting for combat_start.png to appear
[debug] combat_start.png has appeared.
[debug] waiting for combat_start.png to appear
[debug] combat_start.png has appeared.
[log] CLICK on L(936,521)@S(0)[0,0 1360x768]
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[log] CLICK on L(642,322)@S(0)[0,0 1360x768]
[2015-10-01 16:33:12] Spinning compass!
[debug] waiting for P(formation_line_ahead.png) S: 0.99 to appear
[debug] P(formation_line_ahead.png) S: 0.99 has appeared.
[log] CLICK on L(753,261)@S(0)[0,0 1360x768]
[2015-10-01 16:33:25] Selecting fleet formation!
[2015-10-01 16:34:11] Declining night battle!
[debug] waiting for combat_nb_retreat.png to appear
[debug] combat_nb_retreat.png has appeared.
[log] CLICK on L(598,316)@S(0)[0,0 1360x768]
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[log] CLICK on L(1059,508)@S(0)[0,0 1360x768]
[2015-10-01 16:34:27] Checking fleet condition...
[2015-10-01 16:34:36] Light damage: 0; moderate damage: 0; critical damage: 0
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[log] CLICK on L(1059,508)@S(0)[0,0 1360x768]
[debug] waiting for combat_nextnode.png to appear
[debug] combat_nextnode.png has appeared.
[debug] waiting for combat_nextnode.png to appear
[debug] combat_nextnode.png has appeared.
[log] CLICK on L(600,315)@S(0)[0,0 1360x768]
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[log] CLICK on L(642,322)@S(0)[0,0 1360x768]
[2015-10-01 16:35:15] Spinning compass!
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[log] CLICK on L(642,322)@S(0)[0,0 1360x768]
[2015-10-01 16:35:45] Spinning compass!
[debug] waiting for P(formation_line_ahead.png) S: 0.99 to appear
[debug] P(formation_line_ahead.png) S: 0.99 has appeared.
[log] CLICK on L(753,261)@S(0)[0,0 1360x768]
[2015-10-01 16:36:17] Selecting fleet formation!
[error] script stopped with error in line 318
[error] IndexError ( index out of range: 1 )
[error] --- Traceback --- error source first
line: module ( function ) statement 
81: combat (  go_sortie )     if self.night_battles[nodes_run] == 'True':
241: main (  sortie_action )     combat_item.go_sortie()
315: main (  init )     sortie_action()
[error] --- Traceback --- end --------------
```

**Comment by kevin01523** [comment]
kevin@ubuntu:~$ java -jar /home/kevin/SikuliX/sikuli-script.jar -d 2 -r /home/kevin/kancolle-auto/kancolle_auto.sikuli
Picked up JAVA_TOOL_OPTIONS: -javaagent:/usr/share/java/jayatanaag.jar 
[debug] Running on Java 7 (1.7.0_79-b14)
[debug] ResourceLoaderBasic: check: we are running on arch: i386
[debug] ResourceLoaderBasic: check: using Java at: /usr/lib/jvm/java-7-openjdk-i386/jre/
[debug] Screen: initScreens: basic initialization (1 Screen(s) found)
[2015-10-01 19:01:38] Reading config file
['line_ahead', 'line_ahead', 'line_ahead']
['False']
[2015-10-01 19:01:38] Config loaded!
[2015-10-01 19:01:38] Starting kancolle_auto
[2015-10-01 19:01:38] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:01:43] At Home!
[2015-10-01 19:01:43] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:01:47] No, no fleets to receive!
[2015-10-01 19:01:47] Refreshing Home!
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(505,345)@S(0)[0,0 1360x768]
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(377,335)@S(0)[0,0 1360x768]
[2015-10-01 19:01:53] At Home!
[2015-10-01 19:01:53] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:01:57] No, no fleets to receive!
[2015-10-01 19:01:57] Navigating to Expedition menu!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(505,345)@S(0)[0,0 1360x768]
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[log] CLICK on L(979,315)@S(0)[0,0 1360x768]
[debug] waiting for expedition_screen_ready.png to appear
[debug] expedition_screen_ready.png has appeared.
[2015-10-01 19:02:02] Let's send an expedition out!
[debug] waiting for ensei_area_05.png to appear
[debug] ensei_area_05.png has appeared.
[debug] waiting for ensei_area_05.png to appear
[debug] ensei_area_05.png has appeared.
[log] CLICK on L(673,513)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_38.png to appear
[debug] ensei_name_38.png has appeared.
[debug] waiting for ensei_name_38.png to appear
[debug] ensei_name_38.png has appeared.
[log] CLICK on L(602,400)@S(0)[0,0 1360x768]
[debug] waiting for expedition_timer.png to appear
[debug] expedition_timer.png has appeared.
[debug] OCR data path: /usr/local/share/
[debug] TextRecognizer: inited.
[debug] Region.text: #(02:03:21)#
[2015-10-01 19:02:18] Expedition is already running: Expedition 38 (ETA 2015-10-01 21:06:18)
[2015-10-01 19:02:18] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(441,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has appeared.
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has appeared.
[log] CLICK on L(604,370)@S(0)[0,0 1360x768]
[debug] waiting for expedition_timer.png to appear
[debug] expedition_timer.png has appeared.
[debug] Region.text: #(00:39:25)#
[2015-10-01 19:02:32] Expedition is already running: Expedition 5 (ETA 2015-10-01 19:42:32)
[2015-10-01 19:02:32] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(441,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[log] CLICK on L(604,276)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(983,520)@S(0)[0,0 1360x768]
[2015-10-01 19:02:40] Trying to send out fleet 4 for expedition 2
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[log] CLICK on L(758,195)@S(0)[0,0 1360x768]
[2015-10-01 19:02:46] Checking expedition fleet status!
[2015-10-01 19:02:46] Fleet 4 needs resupply!
[debug] waiting for supply_side.png to appear
[debug] supply_side.png has appeared.
[log] CLICK on L(328,284)@S(0)[0,0 1360x768]
[2015-10-01 19:02:47] Lets resupply fleets!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[log] CLICK on L(542,196)@S(0)[0,0 1360x768]
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[log] CLICK on L(424,198)@S(0)[0,0 1360x768]
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[log] CLICK on L(1008,518)@S(0)[0,0 1360x768]
[debug] waiting for supply_not_available.png to appear
[debug] supply_not_available.png has appeared.
[2015-10-01 19:03:01] Done resupplying!
[2015-10-01 19:03:01] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:03:08] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(377,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-10-01 19:03:11] At Home!
[2015-10-01 19:03:11] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:03:15] No, no fleets to receive!
[2015-10-01 19:03:15] Navigating to Expedition menu!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(505,345)@S(0)[0,0 1360x768]
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[log] CLICK on L(979,315)@S(0)[0,0 1360x768]
[debug] waiting for expedition_screen_ready.png to appear
[debug] expedition_screen_ready.png has appeared.
[2015-10-01 19:03:19] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(441,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[log] CLICK on L(604,276)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(983,520)@S(0)[0,0 1360x768]
[2015-10-01 19:03:29] Trying to send out fleet 4 for expedition 2
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[log] CLICK on L(758,195)@S(0)[0,0 1360x768]
[2015-10-01 19:03:35] Checking expedition fleet status!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[log] CLICK on L(931,522)@S(0)[0,0 1360x768]
[debug] waiting for exp_started.png to appear
[debug] exp_started.png has appeared.
[2015-10-01 19:03:45] Expedition sent!: Expedition 2 (ETA 2015-10-01 19:33:10)
[2015-10-01 19:03:47] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:03:55] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(377,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-10-01 19:03:58] At Home!
[2015-10-01 19:03:58] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:04:01] No, no fleets to receive!
[2015-10-01 19:04:01] Navigating to Sortie menu!
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(505,345)@S(0)[0,0 1360x768]
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[log] CLICK on L(539,315)@S(0)[0,0 1360x768]
[debug] waiting for combat_area_2.png to appear
[debug] combat_area_2.png has appeared.
[debug] waiting for combat_area_2.png to appear
[debug] combat_area_2.png has appeared.
[log] CLICK on L(539,517)@S(0)[0,0 1360x768]
[debug] waiting for combat_panel_2-3.png to appear
[debug] combat_panel_2-3.png has appeared.
[debug] waiting for combat_panel_2-3.png to appear
[debug] combat_panel_2-3.png has appeared.
[log] CLICK on L(566,426)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(991,520)@S(0)[0,0 1360x768]
[2015-10-01 19:04:17] Checking fleet condition...
[2015-10-01 19:04:26] Light damage: 0; moderate damage: 0; critical damage: 0
[2015-10-01 19:04:27] Fleet 1 needs resupply!
[2015-10-01 19:04:27] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:04:35] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(377,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-10-01 19:04:37] At Home!
[2015-10-01 19:04:37] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:04:41] No, no fleets to receive!
[2015-10-01 19:04:41] Lets resupply fleets!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for supply_main.png to appear
[debug] supply_main.png has appeared.
[log] CLICK on L(383,300)@S(0)[0,0 1360x768]
[debug] waiting for supply_screen.png to appear
[debug] supply_screen.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[log] CLICK on L(424,198)@S(0)[0,0 1360x768]
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[log] CLICK on L(1008,518)@S(0)[0,0 1360x768]
[debug] waiting for supply_not_available.png to appear
[debug] supply_not_available.png has appeared.
[2015-10-01 19:04:55] Done resupplying!
[2015-10-01 19:04:55] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:05:02] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(377,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-10-01 19:05:05] At Home!
[2015-10-01 19:05:05] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:05:09] No, no fleets to receive!
[2015-10-01 19:05:09] Next sortie!: 2015-10-01 19:03:47
[2015-10-01 19:05:09] Initial checks and commands complete. Starting loop.
[2015-10-01 19:05:09] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:05:14] At Home!
[2015-10-01 19:05:14] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:05:18] No, no fleets to receive!
[2015-10-01 19:05:18] Refreshing Home!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[log] CLICK on L(468,128)@S(0)[0,0 1360x768]
[debug] waiting for top_menu_return.png to appear
[debug] top_menu_return.png has appeared.
[debug] waiting for top_menu_return.png to appear
[debug] top_menu_return.png has appeared.
[log] CLICK on L(363,531)@S(0)[0,0 1360x768]
[2015-10-01 19:05:35] At Home!
[2015-10-01 19:05:35] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-10-01 19:05:39] No, no fleets to receive!
[2015-10-01 19:05:39] Navigating to Sortie menu!
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(505,345)@S(0)[0,0 1360x768]
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[log] CLICK on L(539,315)@S(0)[0,0 1360x768]
[debug] waiting for combat_area_2.png to appear
[debug] combat_area_2.png has appeared.
[debug] waiting for combat_area_2.png to appear
[debug] combat_area_2.png has appeared.
[log] CLICK on L(539,517)@S(0)[0,0 1360x768]
[debug] waiting for combat_panel_2-3.png to appear
[debug] combat_panel_2-3.png has appeared.
[debug] waiting for combat_panel_2-3.png to appear
[debug] combat_panel_2-3.png has appeared.
[log] CLICK on L(566,426)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(991,520)@S(0)[0,0 1360x768]
[2015-10-01 19:05:52] Checking fleet condition...
[2015-10-01 19:06:02] Light damage: 0; moderate damage: 0; critical damage: 0
[2015-10-01 19:06:11] Commencing sortie!
[debug] waiting for combat_start.png to appear
[debug] combat_start.png has appeared.
[debug] waiting for combat_start.png to appear
[debug] combat_start.png has appeared.
[log] CLICK on L(936,521)@S(0)[0,0 1360x768]
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[log] CLICK on L(642,322)@S(0)[0,0 1360x768]
[2015-10-01 19:06:30] Spinning compass!
[debug] waiting for P(formation_line_ahead.png) S: 0.99 to appear
[debug] P(formation_line_ahead.png) S: 0.99 has appeared.
[log] CLICK on L(753,261)@S(0)[0,0 1360x768]
[2015-10-01 19:06:44] Selecting fleet formation!
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[log] CLICK on L(1059,508)@S(0)[0,0 1360x768]
[2015-10-01 19:07:43] Checking fleet condition...
[2015-10-01 19:07:49] Light damage: 1; moderate damage: 0; critical damage: 0
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[debug] waiting for next.png to appear
[debug] next.png has appeared.
[log] CLICK on L(1059,508)@S(0)[0,0 1360x768]
[debug] waiting for next_alt.png to appear
[debug] next_alt.png has appeared.
[debug] waiting for next_alt.png to appear
[debug] next_alt.png has appeared.
[log] CLICK on L(1066,517)@S(0)[0,0 1360x768]
[debug] waiting for combat_nextnode.png to appear
[debug] combat_nextnode.png has appeared.
[debug] waiting for combat_nextnode.png to appear
[debug] combat_nextnode.png has appeared.
[log] CLICK on L(600,315)@S(0)[0,0 1360x768]
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[log] CLICK on L(642,322)@S(0)[0,0 1360x768]
[2015-10-01 19:08:35] Spinning compass!
[debug] waiting for P(formation_line_ahead.png) S: 0.99 to appear
[debug] P(formation_line_ahead.png) S: 0.99 has appeared.
[log] CLICK on L(753,261)@S(0)[0,0 1360x768]
[2015-10-01 19:08:48] Selecting fleet formation!
[error] script stopped with error in line 342
[error] IndexError ( index out of range: 1 )
[error] --- Traceback --- error source first
line: module ( function ) statement 
81: combat (  go_sortie )     if self.night_battles[nodes_run] == 'True':
241: main (  sortie_action )     combat_item.go_sortie()
[error] --- Traceback --- end --------------
kevin@ubuntu:~$

**Comment by mrmin123** [comment]
What's your config.ini look like?

**Comment by mrmin123** [comment]
Is this happening consistently? Are you running on VM?

I won't be able to take a look at the code for a few days but I need some additional info to have an idea of the scope.

**Comment by kevin01523** [comment]
yes im running a vm and it happens kinda commonly only if i do orel 

```
[General]
# The program/window name of your Kancolle container. Examples include Google Chrome,
#  Firefox, KanColleViewer, KanColleTool, etc...
#Program: Google Chrome
Program: KanColleTool Viewer
[Expeditions]
# Define which Expedition a fleet should go on. If you do not have a Fleet unlocked
# or you don't want it to run an expedition, just leave its value blank.
Fleet2: 5
Fleet3: 38
Fleet4: 2

[Combat]
# Set to True if you want Fleet 1 to run combat sorties; False if not.
Enabled: True

# Set which area you want to sortie to.
# Example 1: if you're running 3-2-A, set this to '3'
# Example 2: if you're running 2-3 (Orel), set this to '2'
Area: 2

# Set which subarea you want to sortie to.
# Example 1: if you're running 3-2-A, set this to '2'
# Example 2: if you're running 2-3 (Orel), set this to '3'
Subarea: 3

# Maximum number of combat nodes to run. Resource/hazard nodes do not count.
# Example 1: if you're running 3-2-A, set this to '1' (1st node only)
# Example 2: if you're running 2-3 (Orel), set this to '3' or above
Nodes: 3

# Formations to use at each combat node, separated by commas. Valid options are
# line_ahead, double_line, diamond, echelon, or line_abreast. If the number of
# formations you've specified does not match the number of nodes you specified above,
# the script will default to line_ahead for the remaining nodes.
# Example 1: if you're running 3-2-A, set this to 'line_ahead'
# Example 2: if you're running 2-3 (Orel), set this to 'line_ahead' (remaining nodes
#     defaults to 'line_ahead')
Formations: line_ahead

# Whether or not to engage in night battle at each combat node, separated by commas.
# Valid options are True (to engage in night battle) and False (to avoid night battle).
# If the number of night battle options you've specified does not match the number of
# nodes you specified above, the script will default to True for the remaining nodes.
# Example 1: if you're running 3-2-A, set this to 'False'
# Example 2: if you're running 2-3 (Orel), set this to 'False' (remaining nodes defaults
#     to 'True')
NightBattles: False

# Set the damage threshold for your ships. kancolle-auto will not sortie and instead
# repair if a ship is at this level of damage or below. 0 = light damage, 1 = moderate
# damage, 2 = critical damage.
DamageLimit: 1

# Set the repair time limit for repairs. kancolle-auto will automatically use a bucket
# to repair a ship if its repair time is above this # of hours. Integer values only,
# please. If you want it to never use buckets, set this to something really high like 99.
# If you always want to use buckets, set it to '0'.
RepairTimeLimit: 1
```

**Comment by mrmin123** [comment]
Finally got a chance to look at the code and found the issue... replace this line in `kancolle_auto.py`:

``` python
settings['night_battles'].extend(['True'] * (settings['nodes'] - len(settings['formations'])))
```

with

``` python
settings['night_battles'].extend(['True'] * (settings['nodes'] - len(settings['night_battles'])))
```

A formal fix for it will be in the PR for #22

**Comment by mrmin123** [comment]
@kevin01523 I just pushed the work-in-progress branch [22_repairs](/mrmin123/kancolle-auto/tree/22_repairs) if you want to take a look at it for the above fix and more..
